spec/system/claims/schools/claims/change_claim_on_check_page_spec.rb

```diff
@@ -63,6 +64,19 @@
     then_i_get_a_claim_reference(claim)
   end
 
+  scenario "Anne changes the mentors on claim without inputting hours" do
+    given_i_visit_claim_check_page
+    when_i_click_change_mentors
+    then_i_expect_the_mentors_to_be_checked([mentor1, mentor2])
+    when_i_check_the_mentor(mentor3)
+    when_i_click("Continue")
+    when_i_click("Back")
+    when_i_click("Back")
+    when_i_click("Back")
+    when_i_click("Back")
```

Comment:
Can we add in some test cases to check what the current page is?

i.e.

```
when_i_click "Back"
then_i_should_see_some_page
when_i_click "Back"
then_i_should_see_some_other_page
```

It helps provide the context of what we're clicking "Back" from.